safety/cli.py

```diff
@@ -68,14 +68,15 @@ def check(key, db, json, full_report, bare, stdin, files, cache, ignore, output,
         ]    
     proxy_dictionary = get_proxy_dict(proxyprotocol, proxyhost, proxyport)
     try:
-        vulns = safety.check(packages=packages, key=key, db_mirror=db, cached=cache, ignore_ids=ignore, proxy=proxy_dictionary)
+        vulns, db_last_update = safety.check(packages=packages, key=key, db_mirror=db, cached=cache, ignore_ids=ignore, proxy=proxy_dictionary)
```

Comment:
I am afraid that this is changing our internal code API for Safety. People might be relying on this call and will get unexpected tuple as result instead of a single value. Is there a way to handle this without a breaking change?